lib/jsdom/living/xhr/FormData-impl.js

```diff
@@ -170,9 +169,9 @@ function constructTheFormDataSet(form, submitter) {
 }
 
 function isBlob(obj) {
-  return obj !== null && typeof obj === "object" && blobSymbols.buffer in obj;
+  return obj instanceof Blob;
```

Comment:
This is not correct, as `Object.create(Blob.prototype)` now passes, whereas it should not.